src/docker/docker.md

```diff
@@ -38,4 +38,8 @@ Afterwards, we will look at orchestrating multiple docker images to form a full
 ## Prerequisites
 
 - ✏️ This course only requires Docker Desktop installed. Follow the official docs for [installation steps](https://docs.docker.com/get-docker/).
-- ✏️ This guide uses NodeJS. However, the code is all explained. If you want to run the code without Docker, you will need to [install NodeJS](https://nodejs.org/en/download/). This is optional.
\ No newline at end of file
+- ✏️ This guide uses NodeJS. However, the code is all explained. If you want to run the code without Docker, you will need to [install NodeJS](https://nodejs.org/en/download/). This is optional.
+
+
```

Comment:
My interpretation of the acceptance criteria was that it was supposed to directly mimic React, including the heading.

Could you add a 

`## Next Steps`

here?

and then a dash on the "Head over to" line?

Reply:
React has this:

## Next Steps

✏️ Head over to the [first lesson](learn-react/setting-up-your-environment.html) and get your environment setup.